x/stake/types/msg_test.go

```diff
@@ -171,7 +171,7 @@ func TestMsgBeginRedelegate(t *testing.T) {
 	}
 
 	for _, tc := range tests {
-		msg := NewMsgBeginRedelegate(tc.delegatorAddr, tc.validatorSrcAddr, tc.validatorDstAddr, tc.sharesAmount)
+		msg := NewMsgRedelegate(tc.delegatorAddr, tc.validatorSrcAddr, tc.validatorDstAddr, tc.sharesAmount)
 		if tc.expectPass {
 			require.Nil(t, msg.ValidateBasic(), "test: %v", tc.name)
```

Comment:
I think this PR forget to register the fee parameters for the new transaction types?

Reply:
I did not find the time accumulated part, so it is not implemented right?

Reply:
> I think this PR forget to register the fee parameters for the new transaction types?

you mean like this <https://github.com/bnb-chain/node/blob/6e87245f5adfff1f0e1967a66cf887b3705a3e8e/app/app.go#L333>?

> I did not find the time accumulated part, so it is not implemented right?

right, not implemented yet.